tests/JustSaying.Extensions.DependencyInjection.AwsCore.Tests/JustSaying.Extensions.DependencyInjection.AwsCore.Tests.csproj

```diff
@@ -0,0 +1,44 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);CA1034;CA1051;CA1054;CA1707;CA2000;CA2007</NoWarn>
+    <TargetFramework>net8.0</TargetFramework>
+    <Deterministic>false</Deterministic>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.SimpleNotificationService" VersionOverride="3.7.300.5" />
+    <PackageReference Include="AWSSDK.SQS" VersionOverride="3.7.300.5" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" />
+    <PackageReference Include="GitHubActionsTestLogger" />
+    <PackageReference Include="MELT" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="Polly.Core" />
+    <PackageReference Include="xunit" />
+    <PackageReference Include="xunit.runner.visualstudio" />
+  </ItemGroup>
```

Comment:
I feel you don't need all of these and there's an element of copy-pasta here. Remove the references that aren't actually needed.